.github/workflows/protolint.yaml

```diff
@@ -0,0 +1,19 @@
+name: protolint
+
+on:
+  push:
```

Comment:
For me it looks like we don't need to run actions `when code is pushed to any branch in this repository` push can we add the push later? if required.